src/pages/dataTableUtilities/getColumns.js

```diff
@@ -261,6 +272,14 @@ const COLUMNS = () => ({
     sortable: true,
     editable: false,
   },
+  [COLUMN_NAMES.PAYMENT_NAME]: {
+    type: COLUMN_TYPES.STRING,
+    key: COLUMN_KEYS.OTHER_PARTY_NAME,
+    title: tableStrings.name,
+    alignText: 'left',
+    sortable: false,
+    editable: false,
```

Comment:
Nothing sortable, editable at the moment. Another UI thing to be updated after core functionality done.